autoload/requirements.vim

```diff
@@ -0,0 +1,44 @@
+""
+" @section Introduction, intro
+" @library
+" <doc/@plugin(name).txt> is generated by <https://github.com/google/vimdoc>.
+" See <README.md> for more information about installation and screenshots.
+
+""
+" Update cache.
+function! requirements#update_cache() abort
+    let s:items = []
+    let l:pip_items = split(system('COMP_WORDS="pip install -" COMP_CWORD=2 PIP_AUTO_COMPLETE=1 pip'))
+    let l:pypi_items = split(system('pip-cache pkgnames'))
+    for l:item in l:pip_items
+        let s:items += [{'word': l:item, 'menu': 'pip'}]
+    endfor
+    for l:item in l:pypi_items
+        let s:items += [{'word': l:item, 'menu': 'pypi'}]
+    endfor
+    call writefile([json_encode(s:items)], s:cache)
```

Comment:
pypi_items was not generated in my environment.

Do I install and use this PyPI package myself?
https://pypi.org/project/pip-cache/

pip_items is generated and option names are completed 👍

Reply:
Right, `pip-cache` should be installed. I'll add this information to document.